### PR TITLE
#5139 Keep all error messages from API in Event log during restarting workspace

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Augthors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5139 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Progress/LogProgress.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ import { VirtualizedList } from 'components/common/List';
 import LogRow from './LogRow';
 import * as s from './LogPropessStyles';
 import { DeleteDialog } from 'components/common/Dialog';
+import * as hooks from 'hooks';
+import { useEventLogContentDialog } from 'context';
+import EventLogContentDialog from 'components/EventLog/EventLogContentDialog';
```
Comment (saivirtue) [inline ohara-manager/client/src/components/common/Progress/LogProgress.js:40]: Is this PR has some relation with #5139?
IIRC, these lines are also changes in another PR...

Comment (wu87988622) [reply ohara-manager/client/src/components/common/Progress/LogProgress.js:40]: yeh i will wait #5204 merge.

File: ohara-manager/client/src/components/common/Progress/LogProgress.js [hunk 2]
```diff
@@ -67,6 +69,9 @@ const LogProgress = (props) => {
     isRollbackConfirmDialogOpen,
     setIsRollbackConfirmDialogOpen,
   ] = useState(false);
+  const log = hooks.useLogProgress().data;
+  const { open: openEventLogContentDialog } = useEventLogContentDialog();
+  const handleRowClick = (rowData) => openEventLogContentDialog(rowData);
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Progress/LogProgress.js:74]: abc

**Comment by konekoya** [comment]
retry

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**